modules/cats-effect/src/main/tut/README.md

```diff
@@ -23,17 +23,21 @@ import java.nio.charset.StandardCharsets
 val somePath = Files.createTempFile("config", ".properties")
 val fileContents = "somefield=1234\nanotherfield=some string"
 Files.write(somePath, fileContents.getBytes(StandardCharsets.UTF_8))
+
+implicit val ioCS = cats.effect.IO.contextShift(scala.concurrent.ExecutionContext.global)
 ```
 
 ```tut:silent
 import pureconfig._
 import pureconfig.generic.auto._
 import pureconfig.module.catseffect.syntax._
-import cats.effect.IO
+import cats.effect.{ Blocker, IO }
 
-case class MyConfig(somefield: Int, anotherfield: String)
+final case class MyConfig(somefield: Int, anotherfield: String)
 
-val load: IO[MyConfig] = ConfigSource.file(somePath).loadF[IO, MyConfig]
+val load: IO[MyConfig] = Blocker[IO].use { blocker =>
+  ConfigSource.file(somePath).loadF[IO, MyConfig](blocker)
```

Comment:
Again, we shouldn't teach people to allocate a `Blocker` for a single operation. Same with `save` later.